mappings/net/minecraft/client/render/entity/model/EntityModelSign.mapping

```diff
@@ -0,0 +1 @@
+CLASS none/blb net/minecraft/client/render/entity/model/EntityModelSign
```

Comment:
It's a model for a block *entity* — I was going to make the same comment until I made this observation.